chapters/ch07/2.semantic-search.ipynb

```diff
@@ -509,17 +507,28 @@
    "cell_type": "code",
    "execution_count": 15,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "    def popularity(self, query, position):\n",
```

Comment:
Looks like this notebook's output was generated with OpenSearch instead of Solr. Need Solr output to be the default.

Reply:
7.2 has been re-run w/ Solr